test/Driver/sdk.swift

```diff
@@ -1,14 +1,50 @@
-// RUN: %swiftc_driver -driver-print-jobs -g -sdk %S/../Inputs/clang-importer-sdk %s 2>&1 | FileCheck %s
-// RUN: env SDKROOT=%S/../Inputs/clang-importer-sdk %swiftc_driver_plain -g -driver-print-jobs %s 2>&1 | FileCheck %s
-
-// CHECK-NOT: warning: no such SDK:
-// CHECK: bin/swift
-// CHECK: Driver/sdk.swift
-// CHECK: -sdk {{.*}}/Inputs/clang-importer-sdk
-// CHECK-NEXT: bin/swift
-// CHECK: -sdk {{.*}}/Inputs/clang-importer-sdk
-// CHECK: bin/{{.+}} {{.*}}.o{{[ "]}}
-// CHECK: {{-syslibroot|--sysroot}} {{.*}}/Inputs/clang-importer-sdk
+// RUN: %swiftc_driver -driver-print-jobs -target x86_64-apple-macosx10.9  -g -sdk %S/../Inputs/clang-importer-sdk %s 2>&1 | FileCheck %s --check-prefix OSX
+// RUN: %swiftc_driver -driver-print-jobs -target x86_64-unknown-linux-gnu -g -sdk %S/../Inputs/clang-importer-sdk %s 2>&1 | FileCheck %s --check-prefix LINUX 
+// RUN: %swiftc_driver -driver-print-jobs -target x86_64-unknown-freebsd   -g -sdk %S/../Inputs/clang-importer-sdk %s 2>&1 | FileCheck %s --check-prefix FREEBSD
+
+// RUN: env SDKROOT=%S/../Inputs/clang-importer-sdk %swiftc_driver_plain -target x86_64-apple-macosx10.9  -g -driver-print-jobs %s 2>&1 | FileCheck %s --check-prefix OSX-ENV
+// RUN: env SDKROOT=%S/../Inputs/clang-importer-sdk %swiftc_driver_plain -target x86_64-unknown-linux-gnu -g -driver-print-jobs %s 2>&1 | FileCheck %s --check-prefix LINUX-ENV
+// RUN: env SDKROOT=%S/../Inputs/clang-importer-sdk %swiftc_driver_plain -target x86_64-unknown-freebsd   -g -driver-print-jobs %s 2>&1 | FileCheck %s --check-prefix FREEBSD-ENV
+
+// OSX-NOT: warning: no such SDK:
+// OSX: bin/swift
+// OSX: Driver/sdk.swift
+// OSX: -sdk {{.*}}/Inputs/clang-importer-sdk
+// OSX: bin/{{.+}} {{.*}}.o{{[ "]}}
+// OSX: {{-syslibroot|--sysroot}} {{.*}}/Inputs/clang-importer-sdk
+// OSX-ENV: bin/swift
+// OSX-ENV: Driver/sdk.swift
+// OSX-ENV: -sdk {{.*}}/Inputs/clang-importer-sdk
+// OSX-ENV: bin/{{.+}} {{.*}}.o{{[ "]}}
+// OSX-ENV: {{-syslibroot|--sysroot}} {{.*}}/Inputs/clang-importer-sdk
+
+// LINUX-NOT: warning: no such SDK:
+// LINUX: bin/swift
+// LINUX: Driver/sdk.swift
+// LINUX: -sdk {{.*}}/Inputs/clang-importer-sdk
+// LINUX: bin/{{.+}} {{.*}}swift_begin.o
+// LINUX: {{-syslibroot|--sysroot}} {{.*}}/Inputs/clang-importer-sdk
+// LINUX: {{.*}}swift_end.o
+// LINUX-ENV: bin/swift
```

Comment:
Please use the same check lines for the `-sdk` and `env` tests. The main point of this test is that they should have identical behavior.